### ISSUE TITLE
ansible provisioner hangs indefinitely

### ISSUE THREAD

**Issue by whboyd** [hashicorp/packer#5185]
I'm building a RedHat 7 virtualbox image using an iso, and I am trying to use the ansible provisioner to run some ansible automation against it. However, as soon as the playbook tries to execute a task the build just sits there and hangs indefinitely. 

The output when it hangs is this:
```
rhel73: <127.0.0.1> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o Port=65451 -o 'IdentityFile="/private/var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/ansible-key250038897"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=ansible -o ConnectTimeout=10 -o ControlPath=/Users/whboyd/.ansible/cp/c3be274178 127.0.0.1 '/bin/sh -c '"'"'echo ~ && sleep 0'"'"''
```

I have tried running that ssh command as well as the ansible-playbook command against the running vm while packer was hanging and got the the same result (indefinite hang).

Is anyone able to provide me with some guidance?

packer json:
```
{
  "variables": {
    "ansible_host": "localhost",
    "ansible_connection": "ssh"
  },
  "builders": [{
    "name": "rhel73",
    "type": "virtualbox-iso",
    "headless": "true",
    "guest_os_type": "RedHat_64",
    "iso_url": "/Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso",
    "iso_checksum": "34a65dbdfb8d9bb19b3a03d278df2a99",
    "iso_checksum_type": "md5",
    "ssh_username": "root",
    "ssh_password": "redhat",
    "shutdown_command": "echo 'packer' | sudo -S shutdown -P now",
    "floppy_files": [
      "ks.cfg"
    ],
    "boot_command": [
      "<tab> inst.ks=hd:fd0:/ks.cfg<enter><wait10><esc>"
    ],
    "boot_wait": "5s"
  }],
  "provisioners": [{
    "type": "ansible",
    "playbook_file": "/Users/whboyd/code/ansible/ansible-wut/wut.yml",
    "user": "ansible",
    "ansible_env_vars": [
      "ANSIBLE_SCP_IF_SSH=true",
      "ANSIBLE_HOST_KEY_CHECKING=false"
    ],
    "extra_arguments": [
      "-vvvv"
    ]
  }],
  "post-processors": [
    "vagrant"
  ]
}
```

My ansible playbook:
```
- hosts: all
  gather_facts: false
  tasks:
  - name: do something
    shell: "echo hi"
    register: stuff
  - name: debug
    debug:
      mag: "stuff {{ stuff }}"
  - name: upload a file
    copy:
      dest: "/home/ansible/file.txt"
      content: "howdy"
```

for completeness, my kickstart file:
```
#

auth --enableshadow --passalgo=sha512
reboot --eject
cdrom
text
firstboot --disable
ignoredisk --only-use=sda
keyboard us
lang en_US.UTF-8

network --bootproto=dhcp --onboot=on
services --enabled=network,sshd

rootpw --iscrypted ****
skipx
timezone --utc Etc/GMT

bootloader --location=mbr --driveorder=sda --iscrypted --password=***
clearpart --drives=sda --all
zerombr

part /boot --fstype="ext4" --ondisk=sda --size=512
part pv.22 --fstype="lvmpv" --ondisk=sda --size=19488 --grow

volgroup rhel --pesize=4096 pv.22

logvol swap --fstype="swap" --size=2048 --name=memswap --vgname=rhel --grow
logvol / --fstype="ext4" --size=7200 --name=root --vgname=rhel
logvol /home --fstype="ext4" --fsoptions="defaults,nodev,nosuid" --size=2048 --name=home --vgname=rhel
logvol /tmp --fstype="ext4" --fsoptions="defaults,nodev,nosuid,noexec" --size=2048 --name=tmp --vgname=rhel
logvol /var --fstype="ext4" --fsoptions="defaults,nodev,nosuid" --size=2048 --name=var --vgname=rhel
logvol /var/log --fstype="ext4" --fsoptions="defaults,nodev,nosuid,noexec" --size=2048 --name=var_log --vgname=rhel
logvol /var/log/audit --fstype="ext4" --fsoptions="defaults,nodev,nosuid,noexec" --size=2048 --name=var_log_audit --vgname=rhel

%packages
@core
libselinux-python
%end

#vagrant user
user --name=vagrant --password=*** --gecos="Vagrant default user"
```

and of course the full log:
```
PACKER_LOG=1 packer build vagrant-rhel73.json
2017/07/27 12:11:52 [INFO] Packer version: 1.0.3
2017/07/27 12:11:52 Packer Target OS/Arch: darwin amd64
2017/07/27 12:11:52 Built with Go Version: go1.8.3
2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 Using internal plugin for azure-arm
2017/07/27 12:11:52 Using internal plugin for digitalocean
2017/07/27 12:11:52 Using internal plugin for hyperv-iso
2017/07/27 12:11:52 Using internal plugin for null
2017/07/27 12:11:52 Using internal plugin for openstack
2017/07/27 12:11:52 Using internal plugin for virtualbox-ovf
2017/07/27 12:11:52 Using internal plugin for vmware-iso
2017/07/27 12:11:52 Using internal plugin for amazon-ebs
2017/07/27 12:11:52 Using internal plugin for amazon-ebssurrogate
2017/07/27 12:11:52 Using internal plugin for amazon-instance
2017/07/27 12:11:52 Using internal plugin for docker
2017/07/27 12:11:52 Using internal plugin for googlecompute
2017/07/27 12:11:52 Using internal plugin for virtualbox-iso
2017/07/27 12:11:52 Using internal plugin for vmware-vmx
2017/07/27 12:11:52 Using internal plugin for alicloud-ecs
2017/07/27 12:11:52 Using internal plugin for amazon-chroot
2017/07/27 12:11:52 Using internal plugin for amazon-ebsvolume
2017/07/27 12:11:52 Using internal plugin for parallels-pvm
2017/07/27 12:11:52 Using internal plugin for qemu
2017/07/27 12:11:52 Using internal plugin for triton
2017/07/27 12:11:52 Using internal plugin for cloudstack
2017/07/27 12:11:52 Using internal plugin for file
2017/07/27 12:11:52 Using internal plugin for oneandone
2017/07/27 12:11:52 Using internal plugin for parallels-iso
2017/07/27 12:11:52 Using internal plugin for profitbricks
2017/07/27 12:11:52 Using internal plugin for ansible-local
2017/07/27 12:11:52 Using internal plugin for powershell
2017/07/27 12:11:52 Using internal plugin for shell-local
2017/07/27 12:11:52 Using internal plugin for windows-restart
2017/07/27 12:11:52 Using internal plugin for ansible
2017/07/27 12:11:52 Using internal plugin for puppet-masterless
2017/07/27 12:11:52 Using internal plugin for shell
2017/07/27 12:11:52 Using internal plugin for chef-client
2017/07/27 12:11:52 Using internal plugin for chef-solo
2017/07/27 12:11:52 Using internal plugin for file
2017/07/27 12:11:52 Using internal plugin for salt-masterless
2017/07/27 12:11:52 Using internal plugin for converge
2017/07/27 12:11:52 Using internal plugin for puppet-server
2017/07/27 12:11:52 Using internal plugin for windows-shell
2017/07/27 12:11:52 Using internal plugin for googlecompute-export
2017/07/27 12:11:52 Using internal plugin for vsphere
2017/07/27 12:11:52 Using internal plugin for docker-import
2017/07/27 12:11:52 Using internal plugin for docker-save
2017/07/27 12:11:52 Using internal plugin for vagrant-cloud
2017/07/27 12:11:52 Using internal plugin for artifice
2017/07/27 12:11:52 Using internal plugin for checksum
2017/07/27 12:11:52 Using internal plugin for docker-push
2017/07/27 12:11:52 Using internal plugin for shell-local
2017/07/27 12:11:52 Using internal plugin for alicloud-import
2017/07/27 12:11:52 Using internal plugin for amazon-import
2017/07/27 12:11:52 Using internal plugin for docker-tag
2017/07/27 12:11:52 Using internal plugin for manifest
2017/07/27 12:11:52 Using internal plugin for vagrant
2017/07/27 12:11:52 Using internal plugin for atlas
2017/07/27 12:11:52 Using internal plugin for compress
2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 Attempting to open config file: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 [WARN] Config file doesn't exist: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[amazon-ebs:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-ebs amazon-chroot:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-chroot qemu:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-qemu cloudstack:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-cloudstack file:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-file azure-arm:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-azure-arm hyperv-iso:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-hyperv-iso null:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-null virtualbox-ovf:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-virtualbox-ovf googlecompute:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-googlecompute vmware-vmx:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-vmware-vmx profitbricks:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-profitbricks openstack:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-openstack amazon-ebssurrogate:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-ebssurrogate amazon-instance:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-instance virtualbox-iso:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-virtualbox-iso amazon-ebsvolume:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-ebsvolume parallels-pvm:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-parallels-pvm oneandone:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-oneandone digitalocean:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-digitalocean vmware-iso:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-vmware-iso docker:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-docker alicloud-ecs:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-alicloud-ecs triton:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-triton parallels-iso:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-parallels-iso] PostProcessors:map[docker-import:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-import docker-save:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-save artifice:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-artifice vagrant-cloud:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-vagrant-cloud amazon-import:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-amazon-import docker-tag:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-tag vsphere:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-vsphere docker-push:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-push alicloud-import:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-alicloud-import atlas:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-atlas googlecompute-export:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-googlecompute-export checksum:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-checksum shell-local:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-shell-local manifest:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-manifest vagrant:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-vagrant compress:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-compress] Provisioners:map[powershell:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-powershell windows-restart:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-windows-restart ansible:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-ansible puppet-masterless:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-puppet-masterless chef-client:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-chef-client ansible-local:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-ansible-local shell-local:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-shell-local shell:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-shell chef-solo:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-chef-solo salt-masterless:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-salt-masterless puppet-server:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-puppet-server file:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-file converge:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-converge windows-shell:/usr/local/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-windows-shell]}
2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 Setting cache directory: /Users/whboyd/vagrant/vagrant-rhel73/packer_cache
2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 Loading builder: virtualbox-iso
2017/07/27 12:11:52 Plugin could not be found. Checking same directory as executable.
2017/07/27 12:11:52 Current exe path: /usr/local/bin/packer
2017/07/27 12:11:52 Creating plugin client for path: /usr/local/bin/packer
2017/07/27 12:11:52 Starting plugin: /usr/local/bin/packer []string{"/usr/local/bin/packer", "plugin", "packer-builder-virtualbox-iso"}
2017/07/27 12:11:52 Waiting for RPC address for: /usr/local/bin/packer
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [INFO] Packer version: 1.0.3
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Packer Target OS/Arch: darwin amd64
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Built with Go Version: go1.8.3
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Attempting to open config file: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [WARN] Config file doesn't exist: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[] PostProcessors:map[] Provisioners:map[]}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Setting cache directory: /Users/whboyd/vagrant/vagrant-rhel73/packer_cache
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 args: []string{"packer-builder-virtualbox-iso"}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin minimum port: 10000
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin maximum port: 25000
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin address: unix /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer-plugin386329325
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Waiting for connection...
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Serving a plugin connection...
2017/07/27 12:11:52 Loading provisioner: ansible
2017/07/27 12:11:52 Plugin could not be found. Checking same directory as executable.
2017/07/27 12:11:52 Current exe path: /usr/local/bin/packer
2017/07/27 12:11:52 Creating plugin client for path: /usr/local/bin/packer
2017/07/27 12:11:52 Starting plugin: /usr/local/bin/packer []string{"/usr/local/bin/packer", "plugin", "packer-provisioner-ansible"}
2017/07/27 12:11:52 Waiting for RPC address for: /usr/local/bin/packer
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [INFO] Packer version: 1.0.3
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Packer Target OS/Arch: darwin amd64
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Built with Go Version: go1.8.3
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Attempting to open config file: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [WARN] Config file doesn't exist: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[] PostProcessors:map[] Provisioners:map[]}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Setting cache directory: /Users/whboyd/vagrant/vagrant-rhel73/packer_cache
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 args: []string{"packer-provisioner-ansible"}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin minimum port: 10000
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin maximum port: 25000
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin address: unix /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer-plugin869704410
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Waiting for connection...
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Serving a plugin connection...
2017/07/27 12:11:52 Loading post-processor: vagrant
2017/07/27 12:11:52 Plugin could not be found. Checking same directory as executable.
2017/07/27 12:11:52 Current exe path: /usr/local/bin/packer
2017/07/27 12:11:52 Creating plugin client for path: /usr/local/bin/packer
2017/07/27 12:11:52 Starting plugin: /usr/local/bin/packer []string{"/usr/local/bin/packer", "plugin", "packer-post-processor-vagrant"}
2017/07/27 12:11:52 Waiting for RPC address for: /usr/local/bin/packer
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [INFO] Packer version: 1.0.3
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Packer Target OS/Arch: darwin amd64
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Built with Go Version: go1.8.3
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Attempting to open config file: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [WARN] Config file doesn't exist: /Users/whboyd/.packerconfig
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[] PostProcessors:map[] Provisioners:map[]}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Setting cache directory: /Users/whboyd/vagrant/vagrant-rhel73/packer_cache
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 args: []string{"packer-post-processor-vagrant"}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Detected home directory from env var: /Users/whboyd
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin minimum port: 10000
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin maximum port: 25000
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Plugin address: unix /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer-plugin564444889
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Waiting for connection...
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Serving a plugin connection...
2017/07/27 12:11:52 ui: rhel73 output will be in this color.
2017/07/27 12:11:52 ui:
2017/07/27 12:11:52 Build debug mode: false
2017/07/27 12:11:52 Force build: false
2017/07/27 12:11:52 On error:
2017/07/27 12:11:52 Preparing build: rhel73
rhel73 output will be in this color.

2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 ansible-playbook version: 2.3.1.0
2017/07/27 12:11:52 Waiting on builds to complete...
2017/07/27 12:11:52 Starting build run: rhel73
2017/07/27 12:11:52 Running builder: virtualbox-iso
2017/07/27 12:11:52 [INFO] (telemetry) Starting builder virtualbox-iso
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 VBoxManage path: /usr/local/bin/VBoxManage
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 VBoxManage --version output: 5.1.22r115126
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 VirtualBox version: 5.1.22
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Found Default Guest Additions ISO: /Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Guest additions URL: file:///Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
2017/07/27 12:11:52 ui: ==> rhel73: Downloading or copying Guest additions
==> rhel73: Downloading or copying Guest additions
2017/07/27 12:11:52 ui:     rhel73: Downloading or copying: file:///Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Acquiring lock to download: file:///Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
    rhel73: Downloading or copying: file:///Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Parsed URL: &url.URL{Scheme:"file", Opaque:"", User:(*url.Userinfo)(nil), Host:"", Path:"/Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso", RawPath:"", ForceQuery:false, RawQuery:"", Fragment:""}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [DEBUG] Using local file: /Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
2017/07/27 12:11:52 ui: ==> rhel73: Downloading or copying ISO
==> rhel73: Downloading or copying ISO
2017/07/27 12:11:52 ui:     rhel73: Downloading or copying: file:///Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Acquiring lock to download: file:///Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso
    rhel73: Downloading or copying: file:///Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Parsed URL: &url.URL{Scheme:"file", Opaque:"", User:(*url.Userinfo)(nil), Host:"", Path:"/Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso", RawPath:"", ForceQuery:false, RawQuery:"", Fragment:""}
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 [DEBUG] Using local file: /Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso
2017/07/27 12:11:52 packer: 2017/07/27 12:11:52 Verifying checksum of /Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso
2017/07/27 12:11:59 ui: ==> rhel73: Creating floppy disk...
==> rhel73: Creating floppy disk...
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Floppy path: /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer114142056
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Initializing block device backed by temporary file
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Formatting the block device with a FAT filesystem...
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Initializing FAT filesystem on block device
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Reading the root directory from the filesystem
2017/07/27 12:11:59 ui:     rhel73: Copying files flatly from floppy_files
    rhel73: Copying files flatly from floppy_files
2017/07/27 12:11:59 ui:     rhel73: Copying file: ks.cfg
    rhel73: Copying file: ks.cfg
2017/07/27 12:11:59 ui:     rhel73: Done copying files from floppy_files
    rhel73: Done copying files from floppy_files
2017/07/27 12:11:59 ui:     rhel73: Collecting paths from floppy_dirs
    rhel73: Collecting paths from floppy_dirs
2017/07/27 12:11:59 ui:     rhel73: Resulting paths from floppy_dirs : []
    rhel73: Resulting paths from floppy_dirs : []
2017/07/27 12:11:59 ui:     rhel73: Done copying paths from floppy_dirs
    rhel73: Done copying paths from floppy_dirs
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Suppressing annoying messages in VirtualBox
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Executing VBoxManage: []string{"setextradata", "global", "GUI/RegistrationData", "triesLeft=0"}
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 stdout:
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 stderr:
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Executing VBoxManage: []string{"setextradata", "global", "GUI/SuppressMessages", "confirmInputCapture,remindAboutAutoCapture,remindAboutMouseIntegrationOff,remindAboutMouseIntegrationOn,remindAboutWrongColorDepth"}
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 stdout:
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 stderr:
2017/07/27 12:11:59 packer: 2017/07/27 12:11:59 Executing VBoxManage: []string{"setextradata", "global", "GUI/UpdateCheckCount", "60"}
2017/07/27 12:12:02 packer: 2017/07/27 12:12:02 stdout:
2017/07/27 12:12:02 packer: 2017/07/27 12:12:02 stderr:
2017/07/27 12:12:02 packer: 2017/07/27 12:12:02 Executing VBoxManage: []string{"setextradata", "global", "GUI/UpdateDate", "1 d, 2018-01-01, stable"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 ui: ==> rhel73: Creating virtual machine...
==> rhel73: Creating virtual machine...
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"createvm", "--name", "packer-rhel73-1501171912", "--ostype", "RedHat_64", "--register"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout: Virtual machine 'packer-rhel73-1501171912' is created and registered.
2017/07/27 12:12:05 packer: UUID: 6e67fa90-1392-4506-a513-68bc44a4b4d8
2017/07/27 12:12:05 packer: Settings file: '/Users/whboyd/VirtualBox VMs/packer-rhel73-1501171912/packer-rhel73-1501171912.vbox'
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"modifyvm", "packer-rhel73-1501171912", "--boot1", "disk", "--boot2", "dvd", "--boot3", "none", "--boot4", "none"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"modifyvm", "packer-rhel73-1501171912", "--cpus", "1"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"modifyvm", "packer-rhel73-1501171912", "--memory", "512"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 ui: ==> rhel73: Creating hard drive...
==> rhel73: Creating hard drive...
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"createhd", "--filename", "output-rhel73/packer-rhel73-1501171912.vdi", "--size", "40000", "--format", "VDI", "--variant", "Standard"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout: Medium created. UUID: c32f53db-367f-481e-9ec0-8e375945fd46
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"storagectl", "packer-rhel73-1501171912", "--name", "IDE Controller", "--add", "ide"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"storageattach", "packer-rhel73-1501171912", "--storagectl", "IDE Controller", "--port", "0", "--device", "0", "--type", "hdd", "--medium", "output-rhel73/packer-rhel73-1501171912.vdi", "--nonrotational", "off", "--discard", "off"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"storageattach", "packer-rhel73-1501171912", "--storagectl", "IDE Controller", "--port", "0", "--device", "1", "--type", "dvddrive", "--medium", "/Users/whboyd/vagrant/rhel-server-7.3-x86_64-dvd.iso"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Not attaching guest additions since we're uploading.
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Looking for available port between 5900 and 6000 on 127.0.0.1
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Trying port: 5991
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"modifyvm", "packer-rhel73-1501171912", "--vrdeaddress", "127.0.0.1", "--vrdeauthtype", "null", "--vrde", "on", "--vrdeport", "5991"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Copying floppy to temp location: /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer189396903/floppy.vfd
2017/07/27 12:12:05 ui: ==> rhel73: Attaching floppy disk...
==> rhel73: Attaching floppy disk...
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"storagectl", "packer-rhel73-1501171912", "--name", "Floppy Controller", "--add", "floppy"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"storageattach", "packer-rhel73-1501171912", "--storagectl", "Floppy Controller", "--port", "0", "--device", "0", "--type", "fdd", "--medium", "/var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer189396903/floppy.vfd"}
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Looking for available communicator (SSH, WinRM, etc) port between 2222 and 4444
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Trying port: 4088
2017/07/27 12:12:05 ui: ==> rhel73: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 4088)
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"modifyvm", "packer-rhel73-1501171912", "--natpf1", "packercomm,tcp,127.0.0.1,4088,,22"}
==> rhel73: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 4088)
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout:
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 ui: ==> rhel73: Starting the virtual machine...
==> rhel73: Starting the virtual machine...
2017/07/27 12:12:05 ui:     rhel73: The VM will be run headless, without a GUI. If you want to
    rhel73: view the screen of the VM, connect via VRDP without a password to
    rhel73: rdp://127.0.0.1:5991
    rhel73: The VM will be run headless, without a GUI. If you want to
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 Executing VBoxManage: []string{"startvm", "packer-rhel73-1501171912", "--type", "headless"}
    rhel73: view the screen of the VM, connect via VRDP without a password to
    rhel73: rdp://127.0.0.1:5991
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stdout: Waiting for VM "packer-rhel73-1501171912" to power on...
2017/07/27 12:12:05 packer: VM "packer-rhel73-1501171912" has been successfully started.
2017/07/27 12:12:05 packer: 2017/07/27 12:12:05 stderr:
2017/07/27 12:12:05 ui: ==> rhel73: Waiting 5s for boot...
==> rhel73: Waiting 5s for boot...
2017/07/27 12:12:10 ui: ==> rhel73: Typing the boot command...
==> rhel73: Typing the boot command...
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Special code '<tab>' found, replacing with: [0f 8f]
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char ' ', code '[39 b9]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'i', code '[17 97]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'n', code '[31 b1]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 's', code '[1f 9f]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 't', code '[14 94]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char '.', code '[34 b4]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'k', code '[25 a5]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 's', code '[1f 9f]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char '=', code '[0d 8d]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'h', code '[23 a3]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'd', code '[20 a0]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char ':', code '[2a 27 aa a7]', shift true
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'f', code '[21 a1]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'd', code '[20 a0]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char '0', code '[0b 8b]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char ':', code '[2a 27 aa a7]', shift true
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char '/', code '[35 b5]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'k', code '[25 a5]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 's', code '[1f 9f]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char '.', code '[34 b4]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'c', code '[2e ae]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'f', code '[21 a1]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Sending char 'g', code '[22 a2]', shift false
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Special code '<enter>' found, replacing with: [1c 9c]
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Special code <wait10> found, will sleep 10 seconds at this point.
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Special code '<esc>' found, replacing with: [01 81]
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "0f"}
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 stdout:
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 stderr:
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "8f"}
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 stdout:
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 stderr:
2017/07/27 12:12:10 packer: 2017/07/27 12:12:10 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "39"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "b9"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "17"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "97"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "31"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "b1"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "1f"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "9f"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "14"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "94"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "34"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "b4"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "25"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a5"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "1f"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "9f"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "0d"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "8d"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "23"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a3"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "20"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a0"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "2a"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "27"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "aa"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a7"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "21"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a1"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "20"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a0"}
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stdout:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 stderr:
2017/07/27 12:12:11 packer: 2017/07/27 12:12:11 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "0b"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "8b"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "2a"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "27"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "aa"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a7"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "35"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "b5"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "25"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a5"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "1f"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "9f"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "34"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "b4"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "2e"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "ae"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "21"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a1"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "22"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "a2"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "1c"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "9c"}
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stdout:
2017/07/27 12:12:12 packer: 2017/07/27 12:12:12 stderr:
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "01"}
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 stdout:
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 stderr:
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 Executing VBoxManage: []string{"controlvm", "packer-rhel73-1501171912", "keyboardputscancode", "81"}
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 stdout:
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 stderr:
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 [INFO] Waiting for SSH, up to timeout: 5m0s
2017/07/27 12:12:22 ui: ==> rhel73: Waiting for SSH to become available...
==> rhel73: Waiting for SSH to become available...
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 [INFO] Attempting SSH connection...
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 reconnecting to TCP connection for SSH
2017/07/27 12:12:22 packer: 2017/07/27 12:12:22 handshaking with SSH
2017/07/27 12:13:22 packer: 2017/07/27 12:13:22 [DEBUG] SSH handshake err: Timeout during SSH handshake
2017/07/27 12:13:29 packer: 2017/07/27 12:13:29 [INFO] Attempting SSH connection...
2017/07/27 12:13:29 packer: 2017/07/27 12:13:29 reconnecting to TCP connection for SSH
2017/07/27 12:13:29 packer: 2017/07/27 12:13:29 handshaking with SSH
2017/07/27 12:14:29 packer: 2017/07/27 12:14:29 [DEBUG] SSH handshake err: Timeout during SSH handshake
2017/07/27 12:14:36 packer: 2017/07/27 12:14:36 [INFO] Attempting SSH connection...
2017/07/27 12:14:36 packer: 2017/07/27 12:14:36 reconnecting to TCP connection for SSH
2017/07/27 12:14:36 packer: 2017/07/27 12:14:36 handshaking with SSH
2017/07/27 12:15:36 packer: 2017/07/27 12:15:36 [DEBUG] SSH handshake err: Timeout during SSH handshake
2017/07/27 12:15:43 packer: 2017/07/27 12:15:43 [INFO] Attempting SSH connection...
2017/07/27 12:15:43 packer: 2017/07/27 12:15:43 reconnecting to TCP connection for SSH
2017/07/27 12:15:43 packer: 2017/07/27 12:15:43 handshaking with SSH
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 handshake complete!
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 opening new ssh session
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 [INFO] agent forwarding enabled
2017/07/27 12:16:04 ui: ==> rhel73: Connected to SSH!
==> rhel73: Connected to SSH!
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 VBoxManage --version output: 5.1.22r115126
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 VirtualBox version: 5.1.22
2017/07/27 12:16:04 ui: ==> rhel73: Uploading VirtualBox version info (5.1.22)
==> rhel73: Uploading VirtualBox version info (5.1.22)
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 opening new ssh session
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 Starting remote scp process:  scp -vt .
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 Started SCP session, beginning transfers...
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 Copying input data into temporary file so we can read the length
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 [DEBUG] scp: Uploading .vbox_version: perms=C0644 size=6
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 SCP session complete, closing stdin pipe.
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 Waiting for SSH session to complete.
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 scp stderr (length 28): Sink: C0644 6 .vbox_version
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 VBoxManage --version output: 5.1.22r115126
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 VirtualBox version: 5.1.22
2017/07/27 12:16:04 ui: ==> rhel73: Uploading VirtualBox guest additions ISO...
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 opening new ssh session
==> rhel73: Uploading VirtualBox guest additions ISO...
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 Starting remote scp process:  scp -vt .
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 Started SCP session, beginning transfers...
2017/07/27 12:16:04 packer: 2017/07/27 12:16:04 Copying input data into temporary file so we can read the length
2017/07/27 12:16:05 packer: 2017/07/27 12:16:05 [DEBUG] scp: Uploading VBoxGuestAdditions.iso: perms=C0644 size=59445248
2017/07/27 12:16:05 packer: 2017/07/27 12:16:05 SCP session complete, closing stdin pipe.
2017/07/27 12:16:05 packer: 2017/07/27 12:16:05 Waiting for SSH session to complete.
2017/07/27 12:16:05 packer: 2017/07/27 12:16:05 scp stderr (length 44): Sink: C0644 59445248 VBoxGuestAdditions.iso
2017/07/27 12:16:05 packer: 2017/07/27 12:16:05 Running the provision hook
2017/07/27 12:16:05 [INFO] (telemetry) Starting provisioner ansible
2017/07/27 12:16:05 ui: ==> rhel73: Provisioning with Ansible...
==> rhel73: Provisioning with Ansible...
2017/07/27 12:16:06 packer: 2017/07/27 12:16:06 SSH proxy: serving on 127.0.0.1:65451
2017/07/27 12:16:06 ui: ==> rhel73: Executing Ansible: ansible-playbook --extra-vars packer_build_name=rhel73 packer_builder_type=virtualbox-iso -i /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer-provisioner-ansible627549468 /Users/whboyd/code/ansible/ansible-wut/wut.yml --private-key /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/ansible-key250038897 -vvvv
==> rhel73: Executing Ansible: ansible-playbook --extra-vars packer_build_name=rhel73 packer_builder_type=virtualbox-iso -i /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/packer-provisioner-ansible627549468 /Users/whboyd/code/ansible/ansible-wut/wut.yml --private-key /var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/ansible-key250038897 -vvvv
2017/07/27 12:16:07 ui:     rhel73: Using /Users/whboyd/.ansible.cfg as config file
    rhel73: Using /Users/whboyd/.ansible.cfg as config file
2017/07/27 12:16:07 ui:     rhel73: Loading callback plugin default of type stdout, v2.0 from /usr/local/Cellar/ansible/2.3.1.0/libexec/lib/python2.7/site-packages/ansible/plugins/callback/__init__.pyc
    rhel73: Loading callback plugin default of type stdout, v2.0 from /usr/local/Cellar/ansible/2.3.1.0/libexec/lib/python2.7/site-packages/ansible/plugins/callback/__init__.pyc
2017/07/27 12:16:07 ui:     rhel73:
    rhel73:
2017/07/27 12:16:07 ui:     rhel73: PLAYBOOK: wut.yml **************************************************************
    rhel73: PLAYBOOK: wut.yml **************************************************************
2017/07/27 12:16:07 ui:     rhel73: 1 plays in /Users/whboyd/code/ansible/ansible-wut/wut.yml
    rhel73: 1 plays in /Users/whboyd/code/ansible/ansible-wut/wut.yml
2017/07/27 12:16:07 ui:     rhel73:
    rhel73:
2017/07/27 12:16:07 ui:     rhel73: PLAY [all] *********************************************************************
    rhel73: PLAY [all] *********************************************************************
2017/07/27 12:16:07 ui:     rhel73: META: ran handlers
    rhel73: META: ran handlers
2017/07/27 12:16:07 ui:     rhel73:
    rhel73:
2017/07/27 12:16:07 ui:     rhel73: TASK [do something] ************************************************************
    rhel73: TASK [do something] ************************************************************
2017/07/27 12:16:07 ui:     rhel73: task path: /Users/whboyd/code/ansible/ansible-wut/wut.yml:4
    rhel73: task path: /Users/whboyd/code/ansible/ansible-wut/wut.yml:4
2017/07/27 12:16:07 ui:     rhel73: Using module file /usr/local/Cellar/ansible/2.3.1.0/libexec/lib/python2.7/site-packages/ansible/modules/commands/command.py
    rhel73: Using module file /usr/local/Cellar/ansible/2.3.1.0/libexec/lib/python2.7/site-packages/ansible/modules/commands/command.py
2017/07/27 12:16:07 ui:     rhel73: <127.0.0.1> ESTABLISH SSH CONNECTION FOR USER: ansible
    rhel73: <127.0.0.1> ESTABLISH SSH CONNECTION FOR USER: ansible
2017/07/27 12:16:07 ui:     rhel73: <127.0.0.1> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o Port=65451 -o 'IdentityFile="/private/var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/ansible-key250038897"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=ansible -o ConnectTimeout=10 -o ControlPath=/Users/whboyd/.ansible/cp/c3be274178 127.0.0.1 '/bin/sh -c '"'"'echo ~ && sleep 0'"'"''
    rhel73: <127.0.0.1> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o Port=65451 -o 'IdentityFile="/private/var/folders/f_/0gh6r_cs2jdfkytcbd6380v00000gp/T/ansible-key250038897"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=ansible -o ConnectTimeout=10 -o ControlPath=/Users/whboyd/.ansible/cp/c3be274178 127.0.0.1 '/bin/sh -c '"'"'echo ~ && sleep 0'"'"''
```

**Comment by rickard-von-essen** [comment]
Thanks for reporting this. This is a duplicate of #5142, which have been fixed on master. The fix will be included in the next release.

In the mean time you can either build from master or revert to 1.0.2.

**Comment by whboyd** [comment]
In case anyone stumbles across this in the future, I can confirm that this was my issue and was fixed by downgrading to 1.0.2.

Thanks!
